### PR TITLE
Tests for html5 new input types

### PR DESCRIPTION
**Pull request by aaltat**
Tests for #1291

### CODE REVIEW

**Comment by aaltat** [comment]
Travis is having problem with downloading pypy. Need to wait Travis to sort out their problems before this can be merged.

File: atest/acceptance/keywords/input_html5.robot [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+*** Test Cases ***
+Input field type color
+    Input Text    id:color    \#72da19
+    ${value} =    Get Value    id:color
+    Log    ${value}    # Get value does not return the changed value
+
+Input field type date
+    Input Text    id:date    03/01/2019
+    ${value} =    Get Value    id:date
+    Should Be Equal As Strings    ${value}    2019-03-01
```
Comment (rasjani) [inline atest/acceptance/keywords/input_html5.robot:14]: Locale dependant test or just broken ?  Fails for 2019-01-03 != 2019-03-01

Comment (aaltat) [reply atest/acceptance/keywords/input_html5.robot:14]: Oh darn, should have seen that one coming. Should make a fix that the date is converted to same format.

Comment (aaltat) [reply atest/acceptance/keywords/input_html5.robot:14]: Created #1344 to address the problem.

Comment (aaltat) [reply atest/acceptance/keywords/input_html5.robot:14]: Fixed.

**Pull request merged**